addons/api/addon/generated/models/host.js

```diff
@@ -44,9 +44,27 @@ export default class GeneratedHostModel extends BaseModel {
   })
   version;
 
+  @attr({
```

Comment:
It is unnecessary to formally model `attributes`.

Reply:
Understood. I copied this approach from `host-catalog`. Will provide changes to delete it.